app/src/main/java/com/example/cheatchess/MainActivity.kt

```diff
@@ -23,17 +23,15 @@ class MainActivity : AppCompatActivity(), ChessDelegate {
             val positionFEN = "r2q1rk1/ppp2ppp/3bbn2/3p4/8/1B1P4/PPP2PPP/RNB1QRK1 w - - 5 11"
             val depth = "10"
             val mode = "eval"
-            val options = mapOf(Pair("fen",positionFEN), Pair("depth",depth), Pair("mode",mode))
+            val options = mapOf(Pair("fen", positionFEN), Pair("depth", depth), Pair("mode", mode))
 
             val response = try {
                 RetrofitInstance.api.getStockfishEvaluation(options)
-            }
-            catch (e: IOException) {
-                Log.d("MainActivity","IOException")
+            } catch (e: IOException) {
+                Log.d("MainActivity", "IOException")
```

Comment:
Are you watching for these? It may be better to crash than hide the exception and have it not work properly.